server/detectVersion.ts

```diff
@@ -0,0 +1,9 @@
+const rCapiVersion = /^https:\/\/(?:capi.dev\/|deno.land\/x\/capi)@(.+?)\//
+
+export function detectVersion(): string {
+  const version = rCapiVersion.exec(import.meta.url)?.[1]
+  if (!version) {
+    throw new Error("Could not detect version from URL. Please specify via `--version` flag.")
```

Comment:
This should be handled downstream, in the sync command; rather than throwing here, it will probably be nicer to return undefined. Also, there is no `--version` flag.